news/index.rst

```diff
@@ -2,12 +2,13 @@
 News
 ====
 
-*21/09/2020* – Our course on `"Python for Scientific Computing" </training/scip/python-for-scicomp>`__ is ongoing, next session is Wednesday at 10:00 and will be broadcasted live on `CodeRefinery Twitch <https://www.twitch.tv/coderefinery>`__ (no registration needed). Check :doc:`other trainings coming in October and November </training/index>`. Join our :doc:`daily garage </help/garage>` if you have issues to discuss related to computing or data management.
+*21/09/2020* – Our course on `"Python for Scientific Computing" </training/scip/python-for-scicomp>`__ is ongoing, next session is Wednesday at 10:00 and will be broadcasted live on `CodeRefinery Twitch <https://www.twitch.tv/coderefinery>`__ (no registration needed). 
+
+Check `other trainings coming in October and November </training/>`__. Join our `daily garage </help/garage.rst>`__ if you have issues to discuss related to computing or data management.
+
+News archive
+~~~~~~~~~~~~
+
+*15/09/2020* – Our course on `"Python for Scientific Computing" <training/scip/python-for-scicomp>`__ is ongoing, next session is tomorrow. It can also be watched `live on CodeRefinery Twitch <https://www.twitch.tv/coderefinery>`__ if you did not have time to register. 
 
-.. toctree::
-   :maxdepth: 1
 
-   ../training/scip/python-for-scicomp
-   ../training/index
-   ../help/garage
-   ../training/scicomp-announcements-maillist
```

Comment:
I think maybe we should go back to the old toctree here... I'll find it and push to the branch.